packages/ui/src/memberships/components/MemberListItem/Member.tsx

```diff
@@ -26,7 +26,7 @@ interface MemberListItemProps {
 const EMPTY_SEARCH_FILTERS: MemberSearchFilter[] = ['Membership', 'Membership_ID', 'Account_Address']
 
 export const MemberListItem = ({ member, searchFilter }: MemberListItemProps) => {
-  const balance = useBalance(member.controllerAccount)
+  const balance = useBalance(member.controllerAccount) // better root?
```

Comment:
This is out of scope here. You could open an issue about this.
```suggestion
  const balance = useBalance(member.controllerAccount)
```

Although the way I see it,  controller account makes more sense to me here. Because most transaction will target the user controller account. OTOH the root account is more of a security (best kept offline) to recover memberships.